updates for clarity (#6)
Joystream/founding-members: README.md

This repo contains important data about the state of the program, including the rules, current members, guidance on scoring and submissions, and more.

The program is intended to run until the platform is ready to go live on mainnet. However, some changes in structure along the way must be expected.

## Scoring Periods

> A scoring period is a term defined by a starting date and end date. Note down all of your actions that you think will qualify for [points](#scoring-guidance), and submit them after the end of the period.

Reviewer: ```suggestion
A scoring period is a term defined by a published start and end date. Note down all of your actions that you think will qualify for [points](#scoring-guidance), and [submit them](#submission-guidance) after the period has ended.
```

Author: I don't think "published date" makes sense here?

Reviewer: Updated suggestion.

> Within a week of the end of a scoring period, Jsgenesis will review and "grade" all [submissions](#submission-guidance) in the inbox, and update the website with the most recent scores. If anyone reaches the (TBD) threshold to become a Founding Members they are inducted. You can read about the previous and current scoring periods below.

Reviewer: ```suggestion
Within a week of the end of a scoring period, Jsgenesis will review and "grade" all [submissions](#submission-guidance) in the inbox, and update the website with the most recent scores. If anyone reaches the internal threshold to become a Founding Member they will be inducted. You can read about the previous and current scoring periods below.
```

> We are targeting two weeks long scoring periods at the moment, but this may change in the future.

Reviewer: ```suggestion
We are targeting two week long scoring periods at the moment, but this may change in the future.
```

The full rules of the Founding Member Program can be found [here](/RULES.md).

### Current Period

Currently there are two scoring periods running.

> - [Scoring Period #0](/scoring-periods/0.md), which is meant to allow existing community members to get their just "rewards" for actions performed previously. This means all your activities, on the current chain (launched 21st of September 2020) and all "offchain" activities (from the start of Joystream), until the start of Scoring Period #1 qualifies.

Reviewer: ```suggestion
- [Scoring Period #0](/scoring-periods/0.md), which is meant to allow existing community members to get their "just rewards" for actions performed previously. This means all your activities on the current chain (launched 21st of September 2020) and all "offchain" activities (from the genesis of the project), until the start of Scoring Period #1 will qualify.
```
